frontend/src/app/features/forgot-your-username-or-password/username-found/username-found.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { FindUsernameService } from '@core/services/find-username.service';
+
+@Component({
+  selector: 'app-username-found',
+  templateUrl: './username-found.component.html',
+  styleUrls: ['./username-found.component.scss'],
+})
+export class UsernameFoundComponent implements OnInit {
+  public username: string;
+  public isUsernameFound: boolean;
+
+  constructor(private router: Router, private findUsernameService: FindUsernameService) {}
+
+  ngOnInit(): void {
+    this.getUsernameFound();
+    this.isUsernameFound = this.username !== null ? true : false;
```

Comment:
probably we could simplify it as
```suggestion
    this.isUsernameFound = this.username !== null
```